### ISSUE TITLE
Forbid multiline `f` and `t` strings

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#3405]
Code like this should raise:

```python
>>> x = f'{ 1
... }'
```

If you want to use a multiline string use `"""`
Refs #3404

**Comment by mkaraev** [comment]
@sobolevn can I work on this?

**Comment by sobolevn** [comment]
Sure, thank you! 👍

**Comment by sobolevn** [comment]
tstrings are not out yet, so we can start with just fstrings.

**Comment by mkaraev** [comment]
@sobolevn Just to make sure that I'm getting the problem correctly.
We want to forbid the following cases:

```python
>>> x = f'{ 1
... }'
```

```python
>>> x = f"{ 1
... }"
```

And we want to allow:
```python
>>> x = f'''{ 1
... }'''
```

```python
>>> x = f"""{ 1
... }"""
```
Do I get your idea correctly?

**Comment by sobolevn** [comment]
yes :)

**Comment by mkaraev** [comment]
@sobolevn 
Can you please take a look and check if I get the idea of issue correctly?
https://github.com/wemake-services/wemake-python-styleguide/pull/3411
